### ISSUE TITLE
addCue not triggering function calls

### ISSUE THREAD

**Issue by HenrySkup** [processing/p5.js-sound#371]
#### Nature of issue?

- [x] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [x] Other **Sound** 

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [x] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: v0.8.0 April 08, 2019
- Web browser and version: Chrome Version 75.0.3770.100 (Official Build) (64-bit)
- Operating System: MacOSX
- Steps to reproduce this:

As seen in official example: 

http://p5js.org/reference/#/p5.SoundFile/addCue

Hi, I was trying some of my own stuff out, and was noticing that `addCue()` was not triggering any function call.  Went to the documentation page, and saw that it wasn't working there, as well.  

(On a side note: `jump()` was also not working for me... it was ( i think) creating multiple instances of a sound playing ... maybe not properly closing/stopping them? . I'll create a issue about this, too)

**Comment by lucierabahi** [comment]
Same for me. I was trying the addCue() as show in the Coding Train (<https://www.youtube.com/watch?v=SfA5CghXw18&list=PLRqwX-V7Uu6aFcVjlDAkkGIixw70s7jpW&index=3>) and no way to change the background color.

I tried with 2 arguments:
```
function setup() {
  createCanvas(200, 200);
  song = loadSound("end-of-the-world.mp3", loaded);
  button = createButton("play");   
  button.mousePressed(togglePlaying); 
  // background(51);     // I need a draw() function to have a bg,
                         // unlike in the video where he can use it here in setup()
  song.addCue(5, changeBackground);
}
```

```
function changeBackground() {
background(random(255), random(255), random(255));
}
```
and with 3 arguments:
```
function setup() {
  createCanvas(200, 200);
  song = loadSound("end-of-the-world.mp3", loaded);
  button = createButton("play");   
  button.mousePressed(togglePlaying); 

  song.addCue(2, changeBackground, color(0, 0, 255)); 
  song.addCue(4, changeBackground, color(255, 0, 0)); 
  song.addCue(6, changeBackground, color(0, 255, 0)); 
}
```

```
function changeBackground(color) {
  background(color);
}
```

Note that the 
```
currentTime() function
function draw() {
  background(song.currentTime() * 10, 0, 255)
}
```  
on the other hand is working and changes the background color.

**Comment by stalgiag** [comment]
Just wanted to bump this as we have gotten a couple of issue reports related to his on the p5.js repo. I tested the [soundfile_remove_cue example](https://github.com/processing/p5.js-sound/tree/master/examples/soundfile_remove_cue) and it works as expected. Could this be related to the [unresolved bump to 0.3.12](https://github.com/processing/p5.js/pull/3988) in the p5.js repo?

**Comment by therewasaguy** [comment]
thanks for reporting! This should be fixed with #405 and other recent changes...soon to appear on p5js web reference
